app/models/bank_account.rb

```diff
@@ -0,0 +1,12 @@
+class BankAccount < ActiveRecord::Base
+
+  has_many :bank_transactions, dependent: :destroy
+
+  normalize_attributes :name, :iban, :description
+
+  validates :name, :presence => true, :uniqueness => true, :length => { :minimum => 2 }
+  validates :iban, :presence => true, :uniqueness => true
+  validates_format_of :iban, :with => /\A[A-Z]{2}[0-9]{2}[0-9A-Z]{,30}\z/
```

Comment:
Do we need to validate the check digit?

Reply:
we don't do it at any of the other places (using a IBAN checkin library is IMHO a independent point)

Reply:
Agree.